docs/epidata_development.md

```diff
@@ -49,346 +49,125 @@ $ [sudo] make test pdb=1
 $ [sudo] make test test=repos/delphi/delphi-epidata/integrations/acquisition
 ```
 
-## Long version
+You can read the commands executed by the Makefile [here](../dev/local/Makefile).
```

Comment:
this link doesnt work; it ends up pointing to `https://cmu-delphi.github.io/dev/local/Makefile` ... try `https://github.com/cmu-delphi/delphi-epidata/blob/dev/dev/local/Makefile` instead

Reply:
ah, i formatted the link so it would work [here](https://github.com/cmu-delphi/delphi-epidata/blob/628e9655144934f3903c133b6713df4d4fcc613e/docs/epidata_development.md)

Reply:
i think it should work for both if we set it to `https://github.com/cmu-delphi/delphi-epidata/blob/dev/dev/local/Makefile`